_includes/code-of-conduct-short.html

```diff
@@ -0,0 +1,12 @@
+<section id="code_of_conduct">
```

Comment:
I'm being very picky here and it really doesn't matter, but oddly the naming convention for section id's (inherited from the prior code) is to just use lower case without spaces/underscores/hyphens. So this would be:

`<section id="codeofconduct">`